src/styles/blocks/_bike-card.scss

```diff
@@ -0,0 +1,34 @@
+.bike-card {
+  width: 280px;
+
+  @include onTablet {
+    @include cardSize(392px);
+  }
+
+  @include onDesktop {
+    @include cardSize(33%);
+  }
+
+  &__image {
+    width: 100%;
+
+    transition: transform 0.3s;
```

Comment:
Also, good practice to use variables for transition duration, because you use them in many places.